src/main/scala/DataMunging.scala

```diff
@@ -0,0 +1,40 @@
+/**
+  * Created by idarlington on 2/26/17.
+  */
+object DataMunging {
+
+  case class weatherLine(
+                          dayNumber : Int,
+                          maxTemperature : Double,
+                          minTemperature : Double
+                        )
+
+  def temperatureDiff (a : weatherLine, b : weatherLine) = {
+    (a.maxTemperature - a.minTemperature) > (b.maxTemperature - b.minTemperature)
+  }
+
+  def main(args: Array[String]) = {
+    /** Read Contents of file **/
+    val filename = "weather.dat"
+    val fileLines = io.Source.fromFile(filename).getLines().toList
+
+    /** filter empty lines **/
```

Comment:
Your solution is nice, simple, elegant. 👍 

The comments would be really helpful for guys in our Meetup who are beginners. Though more experienced Scala devs may not need them, as the code itself is quite clear. 👍